src/localization/en/settings.ts

```diff
@@ -52,6 +52,11 @@ export default {
             confirm: 'Nice!',
         },
     },
+    licenses: {
+        formlist: {
+            license: 'License',
```

Comment:
if the duplication here were gone, it would be perfect 💜

Reply:
![image](https://github.com/neuland-ingolstadt/neuland.app-native/assets/39560569/67337e53-7d15-4adc-bebd-eb262e4f26ee)